### ISSUE TITLE
[FEATURE REQUEST] MK4 LCD display showing remaining time to next color change

### ISSUE THREAD

**Issue by gigi84** [prusa3d/Prusa-Firmware-Buddy#3429]
**Printer type** - [MK4]

Additionally to the total remaining time, I would like to see on the LCD display (and PrusaLink) the remaining time to the next color change

**Comment by codingcatgirl** [comment]
This would be great!

**Comment by schorsch3000** [comment]
also include pauses in that, please :-)

**Comment by Claghorn** [comment]
Yep. Time to anything that requires human interaction would be very handy.

**Comment by Pdamga** [comment]
👍

**Comment by gdbarron** [comment]
Duplicate of #3107?

**Comment by metebalci** [comment]
+1

**Comment by gegy1** [comment]
Yes, this would be very useful for the XL when adding a pause in the slicer. The printer (and also PrusaConnect) should show the time when the pause will be reached. (Remaining time and clock time)

**Comment by danopernis** [comment]
@gegy1 this should be already fixed in the firmware since at least v6.0.0, can you please confirm that?

**Comment by gegy1** [comment]
Yes your are right, i can see the remaining time for next filament change. What i mean is, that i do not want to see the remaining time, i want to so the time like "Next filament change (pause) at 7:32pm" :)

![image](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/75700881/b0178c1c-2573-4d35-a9cb-22cbf3b16fb9)

**Comment by nomike** [comment]
While the remaining time until the next filament change is shown in PrusaConnect, it would be awesome if this were also displayed in PrusaLink.

**Comment by schorsch3000** [comment]
This issue issn't stale because interst is dropped, but because there are no open questions and it is not worked on (which is fine, but closing because its not worked on is not)

**Comment by schorsch3000** [comment]
so apparently no one worked on that feature request for 60 days and somehow by that interst in that feature vanished :-D
